airflow/serialization/serializers/timezone.py

```diff
@@ -29,10 +30,15 @@
 serializers = [
     "pendulum.tz.timezone.FixedTimezone",
     "pendulum.tz.timezone.Timezone",
-    "zoneinfo.ZoneInfo",
-    "backports.zoneinfo.ZoneInfo",
 ]
 
+PY39 = sys.version_info >= (3, 9)
+
+if PY39:
+    serializers.append("zoneinfo.ZoneInfo")
+else:
+    serializers.append("backports.zoneinfo.ZoneInfo"),
```

Comment:
Static checks are complaining about that

```suggestion
    serializers.append("backports.zoneinfo.ZoneInfo")
```